.env.template

```diff
@@ -7,3 +7,9 @@ FLASK_DEBUG=true
 
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
+
+TRELLO_API_KEY=trello-api-key
+TRELLO_SECRET=trello-secret
+TRELLO_BOARD_ID=trello-board-id
+TRELLO_TODO_LIST_ID=trello-todo-list-id
+TRELLO_COMPLETED_LIST_ID=trello-completed-list-id
```

Comment:
Nice, thanks for keeping this updated - it makes my life much easier to test out your app, which is a good sign that it would be really helpful for future/other developers (which includes yourself coming back after a break!)